cpp/src/backends/torch_scripted/handler/base_handler.cc

```diff
@@ -172,7 +172,7 @@ namespace torchserve {
             200,
             "data_type",
             torchserve::PayloadType::kDATA_TYPE_BYTES,
-            torch::pickle_save(data[kv.first]));
+            torch::pickle_save(msg));
```

Comment:
msg should be removed. I will submit PR for the fixing.

Reply:
Will refactor

Reply:
done